### PR TITLE
Add docs for updating COVID Local Restriction Data

### PR DESCRIPTION
**Pull request by Rosa-Fox**

### CODE REVIEW

File: docs/update-covid-local-restriction-data.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+Otherwise we just display the current restrictions. This is determined by the `start_date` and `start_time`.
+
+![Image of results page without changing alert level](docs/alert-level-no-upcoming-changes.png)
+
+Example of somewhere that just has current restrictions:
+
+```
+E08000012:
+  name: Liverpool City Council
```
Comment (leenagupte) [inline docs/update-covid-local-restriction-data.md:30]: One thing we should mention is that we use Mapit to look up the "name" we enter here.

**Review by leenagupte — COMMENTED** [review]
One comment. Otherwise looks good!

File: docs/update-covid-local-restriction-data.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    - alert_level: 3
+      start_date: 2020-10-23
+      start_time: "00:01"
+```
+
+To update the restriction data, you will need to [add a list to the YAML](https://github.com/alphagov/collections/blob/master/lib/local_restrictions/local-restrictions.yaml) for the given GSS codes that contain:
+
+- the new `alert_level`.
+- the `start_date` for when the restrictions will be introduced.
+- the `start_time` for when the restrictions will be introduced.
```
Comment (leenagupte) [inline docs/update-covid-local-restriction-data.md:55]: We should also mention, that a "new" entry should be added for the upcoming changes. Almost overwrote some existing ones not 5 minutes ago! 😱

File: docs/update-covid-local-restriction-data.md (hunk 1 above)
Comment (Rosa-Fox) [reply docs/update-covid-local-restriction-data.md:30]: thanks, have added :)

File: docs/update-covid-local-restriction-data.md (hunk 2 above)
Comment (Rosa-Fox) [reply docs/update-covid-local-restriction-data.md:55]: Thanks @leenagupte! have added something about adding rather than overwriting!

**Review by leenagupte — APPROVED** [review]
Looks good! 🎉 ⭐

**Pull request merged**